meeting-notes/2023-06-08-Falco-Core-Maintainers.md

```diff
@@ -0,0 +1,139 @@
+# 2023-06-08 Falco Core Maintainers (monthly)
+
+## MC
+
+- leogr
+
+## Who joined
+
+- Leonardo Grasso ([@leogr](https://github.com/leogr))
+- Melissa Kilby ([@incertum](https://github.com/incertum))
+- Jason Dellaluce([@jasondellaluce](https://github.com/jasondellaluce))
+- Massimiliano Giovagnoli ([@maxgio92](https://github.com/maxgio92))
+- Grzegorz Nosek ([gnosek](https://github.com/gnosek))
+- Andrea
```

Comment:
Hey mom it's me! :rofl: :rofl: BTW we miss the rest of my name and @LucaGuerra here